### ISSUE TITLE
[nginx-mainline] typo in mime.types files

### ISSUE THREAD

**Issue by fmonts** [oerdnj/deb.sury.org#1769]
After updating to the latest nginx mainline, `nginx -t` started showing this warning:

> nginx: [warn] duplicate extension "woff", content type: "font/woff2", previous content type: "font/woff" in /etc/nginx/mime.types:29

After some investigation I found where the issue is, the file `/etc/nginx/mime.types` has: 

    font/woff                             woff;
    font/woff2                            woff;

Instead of: 

    font/woff                                        woff;
    font/woff2                                       woff2;

You can check for reference the file in the Nginx Master branch: https://github.com/nginx/nginx/blob/master/conf/mime.types

I have the same issue on all my 20.04 and 18.04 machines

**Comment by fmonts** [comment]
I manually fixed it, but now there are conflicts in updates:

```
--- /etc/nginx/mime.types       2022-05-03 10:49:44.955802656 +0000
+++ /etc/nginx/mime.types.dpkg-new      2022-04-28 12:37:17.000000000 +0000
@@ -26,7 +26,7 @@
     image/x-ms-bmp                        bmp;

     font/woff                             woff;
-    font/woff2                            woff2;
+    font/woff2                            woff;

     application/java-archive              jar war ear;
     application/json                      json;
```

**Comment by beatquantum** [comment]
I have the same issue on Ubuntu 22.04 Jammy.

"systemctl status nginx"   showed the following:

Jun 19 18:54:29 REDACTED nginx[4291]: nginx: [warn] duplicate extension "woff", content type: "font/woff2", previous content type: "font/woff" in /etc/nginx/mime.types:29
Jun 19 18:54:29 REDACTED nginx[4292]: nginx: [warn] duplicate extension "woff", content type: "font/woff2", previous content type: "font/woff" in /etc/nginx/mime.types:29

I changed the extension manually to woff2, restarted nginx, the error is gone.

**Comment by oerdnj** [comment]
This should have been fixed in the 1.22.0 and 1.23.0 upload. I am now discarding the Debian file and using the upstream `mime.types` file as it is.
